src/main/generated/resources/assets/minecolonies/lang/default.json

```
@@ -32,6 +32,8 @@
   "com.minecolonies.research.effects.tooldurabilitymultiplier.description": "Citizen Tools +%3$s%% Durability",
   "com.minecolonies.research.effects.walkingmultiplier.description": "Citizen Walk Speed +%3$s%%",
   "com.minecolonies.research.effects.workingdayhaddition.description": "Citizen Work Day +%sH",
+  "com.minecolonies.research.effects.resurectchanceaddition.description": "Improve resurection chance +%3$s%%",
```

```suggestion
  "com.minecolonies.research.effects.resurrectchanceaddition.description": "Improve Resurrection Chance by +%3$s%%",
```
These should be title case